app/models/katello/alternate_content_source.rb

```diff
@@ -29,13 +29,18 @@ class AlternateContentSource < Katello::Model
              inverse_of: :alternate_content_source
     has_many :smart_proxies, -> { distinct }, through: :smart_proxy_alternate_content_sources
 
-    validates :base_url, :subpaths, :verify_ssl, :upstream_username,
+    validates :base_url, :subpaths, :upstream_username,
               :upstream_password, :ssl_ca_cert, :ssl_client_cert, :ssl_client_key, if: :simplified?, absence: true
     validates :base_url, if: -> { custom? || rhui? }, presence: true
     validates :products, if: -> { custom? || rhui? }, absence: true
     validates :label, :uniqueness => true
     validates :name, :uniqueness => true, presence: true
-    validates :verify_ssl, if: :custom?, exclusion: [nil]
+    # verify ssl must be validated this way due to presence: <bool> failing on a value of false
+    validates :verify_ssl, if: -> { custom? || rhui? }, inclusion: {
+      in: [true, false],
+      message: "can't be blank"
+    }
+    validates :verify_ssl, if: :simplified?, inclusion: { in: [nil] }
```

Comment:
This results in a message "Verify ssl is not included in the list." can you add a custom message "must be blank" ?

Reply:
Fixed!